test/resque_failure_multiple_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+require 'resque/failure/multiple'
+
+describe "Resque::Failure::Multiple" do
+
+  it 'it requeue_all and does not raise an exception' do
+    with_failure_backend(Resque::Failure::Multiple) do
+      Resque::Failure::Multiple.classes = [Resque::Failure::Redis]
+      exception = StandardError.exception('some error')
+      worker = Resque::Worker.new(:test)
+      queue = "queue"
+      payload = { "class" => Object, "args" => 3 }
+      Resque::Failure.create({:exception => exception, :worker => worker, :queue => queue, :payload => payload})
+      Resque::Failure.requeue_all # should not raise an error
```

Comment:
I was very confused by why the coverage decreased when you clearly added a test for this, but I think I figured it out. Shouldn't this line be `Resque::Failure::Multiple.requeue_all` since that's the method you added?